src/components/DropdownSearch/index.js

```diff
@@ -116,22 +129,18 @@ function DropdownSearch({
       <InputBase
         inputProps={{ "aria-label": "search" }}
         onChange={handleChange}
+        placeholder={variant === "explore" ? label : ""}
```

Comment:
Why not set label to ""  in explore so that we don't need this check here?

Reply:
> explore variant on the drop-down search, pass in proper ic


> Looks good just not sure about the logic of choosing explore navigation. My thinking was explore page will set the explore variant on Navigation which will _use_ `ExploreNavigation` (no need for variant on this component since it's specifically for explore page). `ExploreNavigation` will in turn set explore variant on the drop-down search, pass in proper icons, etc.




@kilemensi explore page is passing a variant explore on its props. Because the difference is only on Desktop Nav, Mobile Nav similar for both explore and non-explore pages. I am doing the check on variant on Navigation Index ..Desktop part

Reply:
Cool, cool